terra2/assetlist.json

```diff
@@ -1781,6 +1781,40 @@
           "provider": "ERIS Protocol"
         }
       ]
+    },
+    {
```

Comment:
You had an extra comma here before which I removed. You also need type_asset (sdk.coins, I believe)